### PR TITLE
[1438] Make the accept transition screen idempotent

### PR DESCRIPTION
**Pull request by davidgisbey**
### Context

If a user tries to accept the transition screen while already rolled over, it currently throws a sentry error. This is undesirable behaviour.

### Changes proposed in this pull request

- Make the accept transition screen idempotent

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: spec/requests/api/v2/users/accept_transition_screen_spec.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def perform_request
       end
 
       it 'returns error' do
-        expect { perform_request }.to raise_error AASM::InvalidTransition
```
Comment (benilovj) [inline spec/requests/api/v2/users/accept_transition_screen_spec.rb]: The spec name no longer matches the spec

**Comment by davidgisbey** [comment]
[Make accept_transition_screen idempotent in mc-be](https://trello.com/c/q3EFc1UD/1438-make-accepttransitionscreen-idempotent-in-mc-be)

File: app/controllers/api/v2/users_controller.rb [hunk 2]
```diff
@@ -18,7 +18,9 @@ def update
       end
 
       def accept_transition_screen
-        @user.accept_transition_screen!
+        if @user.state == 'new'
+          @user.accept_transition_screen!
+        end
       end
 
       def accept_rollover_screen
```
Comment (fofr) [inline app/controllers/api/v2/users_controller.rb]: We should do the same for the rollover screen.

Comment (davidgisbey) [reply app/controllers/api/v2/users_controller.rb]: Cool i'll add that in and edit the card.

File: spec/requests/api/v2/users/accept_transition_screen_spec.rb (hunk 1 above)
Comment (davidgisbey) [reply spec/requests/api/v2/users/accept_transition_screen_spec.rb]: Thanks i'll change it now.

**Pull request merged**